src/server/services/transformers/service-status.js

```diff
@@ -187,9 +191,101 @@ function serviceStatus(service) {
           ?.map((error) => error?.message)
           .filter(Boolean) ?? []
     },
+    cdpSquidProxy: {
+      name: 'Proxy',
+      part: 4,
+      url: {
+        text: `${githubOrg}/cdp-squid-proxy`,
+        href: `https://github.com/${githubOrg}/cdp-squid-proxy`
+      },
+      status: {
+        text: cdpSquidProxy?.status
+          ? startCase(cdpSquidProxy?.status)
+          : unknownValue,
+        classes: statusTagClassMap(cdpSquidProxy?.status)
+      },
+      info: () => {
+        switch (cdpSquidProxy?.status) {
+          case creationStatuses.raised:
+          case creationStatuses.requested:
+          case creationStatuses.inProgress:
+            return `Setting up proxy access.`
+          case creationStatuses.created:
+          case creationStatuses.success:
+            return `Your test suite can make outbound calls via the proxy.`
+          case creationStatuses.unknown:
+          case creationStatuses.failure:
+            return `Something has gone wrong, contact us using the details at the ${buildLink(
+              '#app-help',
+              'top of the page',
+              false
+            )}.`
+          default:
+            return 'Status unknown'
+        }
+      },
+      githubAction: {
+        name: cdpSquidProxy?.main?.workflow?.name,
+        url: {
+          text: removeUrlParts(cdpSquidProxy?.main?.workflow?.html_url),
+          href: cdpSquidProxy?.main?.workflow?.html_url
+        },
+        started: cdpSquidProxy?.main?.workflow?.created_at
+      },
+      errors:
+        cdpSquidProxy?.result?.errors
+          ?.map((error) => error?.message)
+          .filter(Boolean) ?? []
+    },
+    cdpDashboard: {
+      name: 'Dashboards',
+      part: 5,
+      url: {
+        text: `${githubOrg}/cdp-grafana-svc`,
+        href: `https://github.com/${githubOrg}/cdp-grafana-svc`
+      },
+      status: {
+        text: cdpDashboard?.status
+          ? startCase(cdpDashboard?.status)
+          : unknownValue,
+        classes: statusTagClassMap(cdpDashboard?.status)
+      },
+      info: () => {
+        switch (cdpDashboard?.status) {
+          case creationStatuses.raised:
+          case creationStatuses.requested:
+          case creationStatuses.inProgress:
+            return `Setting up dashboards.`
+          case creationStatuses.created:
+          case creationStatuses.success:
+            return `Default dashboards created.`
```

Comment:
doesn't need to be a template literal, no variable so single quotes are fine

Reply:
Also can I suggest:

`Default Grafana observability metrics data and graph dashboards created for your service.`